tests/integration/cloudformation/resources/test_ec2.py

```diff
@@ -61,3 +61,22 @@ def test_cfn_with_multiple_route_table_associations(ec2_client, deploy_cfn_templ
     assert subnet1["EnableDns64"] is True
     assert subnet1["MapPublicIpOnLaunch"] is True
     assert subnet2["PrivateDnsNameOptionsOnLaunch"]["HostnameType"] == "ip-name"
+
+
+@pytest.mark.skip_snapshot_verify(paths=["$..DriftInformation", "$..Metadata"])
+def test_internet_gateway_ref_and_attr(deploy_cfn_template, cfn_client, snapshot):
```

Comment:
Since we're using snapshots here I guess this was also executed against AWS and can be marked as such? (`aws_validated`)